src/server/types.ts

```diff
@@ -33,30 +33,34 @@ export interface IslandModule {
 }
 
 export interface HandlerContext<T = unknown> {
-  req: Request;
   match: Record<string, string>;
```

Comment:
Want to do this rename in a follow up?

Reply:
yea, waas also considering doing #96 in this PR, but maybe its just cleaner to have these things in separate PRs